docker-compose.yml

```diff
@@ -1,27 +1,26 @@
 # This docker-compose file will stand up the entire stack including ingress and inventory.
 version: "3"
 services:
-  zookeeper:
-    image: quay.io/cloudservices/cp-zookeeper
-    environment:
-      - ZOOKEEPER_CLIENT_PORT=32181
-      - ZOOKEEPER_SERVER_ID=1
-      - KAFKA_OPTS=-Dzookeeper.admin.enableServer=false
   kafka:
-    image: quay.io/cloudservices/cp-kafka
+    image: docker.io/confluentinc/cp-kafka
```

Comment:
@Victoremepunto you might not like this, but our mirrored version is a year old without KRaft support. Also mirroring doesn't copy aarch64 images and x86 emulated kafka on my M2 Mac is dying 😓